shared/workspace-cleanup.tf

```diff
@@ -0,0 +1,4 @@
+module "workspace-cleanup" {
+  source  = "github.com/TomTucka/terraform-workspace-manager/terraform/workspace_cleanup"
```

Comment:
Fine for now but are we planning to have this as an MOJ repo?

Reply:
Was thinking of pulling out the table setup and just using it as is but we're using the repo for the executable as well and it's Tom rather than external but yeah you're right, i'll ask him about whether we can move this to moj